recipes/proplot/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "proplot" %}
+{% set version = "0.3.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 75e6c871bb628ed2879eb1a997c159a405ba69655e8010489f50f74877813b7a
+
+build:
+  noarch: python
+  number: 0
+  script:
+    - rm -rf proplot/fonts
```

Comment:
I just saw the developer has commented earlier that this `rm` line is not needed anymore. Do you want to try removing it? I am ok merging the PR regardless.

Reply:
It still vendors quite a few fonts.

We would have to make sure that all those licenses are valid.

I think for now, it is best to simply avoid the fonts folder due to licensing confusion

![image](https://user-images.githubusercontent.com/90008/71549386-6716e200-298a-11ea-9547-0461d028445f.png)